_posts/2022-11-23-trino-summit-2022-zillow-recap.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "Optimizing Trino using spot instances with Zillow"
+author: "Santhosh Venkatraman, Rupesh Kumar Perugu, Brian Olsen"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/zillow.jpg
+---
+
+In the next installment of [the Trino Summit 2022 sessions posts]({% post_url
+2022-11-21-trino-summit-2022-recap %}), we jump into an exciting topic by folks
+from Zillow about running Trino on spot instances. Spot instances are cheap and
+ephemeral nodes that lead to much cheaper compute. Spot instances are cheaper as
+they are not guaranteed to remain available.
+
+In this session, Zillow engineers talk about how they use Trino on spots to take
+advantage of the cost savings while handling the transitory nature of spots.
+
+<!--more-->
+
+{% youtube vz9reBUgQTE %}
+
+<a class="btn btn-pink btn-md" target="_blank" href="/assets/blog/trino-summit-2022/Trino@Zillow.pdf">
+  Check out the slides here!
+</a>
+
+## Recapping
+
+Zillow's BI platform team is tasked with enabling access to data and metrics
+from their data lake in a self-serving and performant manner. The platform must
+handle generating up-to-date reports and metrics to unlock time-critical
+opportunities. They also need to enable adhoc analytics across multiple domains
+within Zillow. 
+
+There are close to 600 data pipelines and 65,000 queries running daily. The
+average read covers 600 terabytes of data, and the average P95 time is around
+20 seconds. They have 6 Trino services that service various workflow based on
+load. These are all deployed on Amazon EKS with a range of eight to 60 workers
+based on CPU utilization.
+
+When deploying Trino on EKS, Zillow uses worker groups, which enables them to
+collocate nodes in AWS local zones. It also made it possible to choose spot 
+instances, which are 90% cheaper than regular on-demand instances. A critical
+aspect they needed to do was correctly tune the percentage of nodes that were
+spot instances, and create pools of nodes that were entirely on-demand
+(coordinators) and some that are an optimal blend of spot and on-demand.
```

Comment:
on-demand for coordinators

Reply:
also .. we do we need to explain why? Coordinator being spot would take the whole cluster down